### PR TITLE
feat: Change global-read-only policy to non editable

### PR DESCRIPTION
**Pull request by valeriia-ruban**
### Description

Update global-read-only to non editable
image
image

### Testing & Reproduction steps

1. set up CONSUL_ACLS_ENABLE to 1 in session storage
2. set CONSUL_POLICY_COUNT = 10
3. go to Policies
4. log in with any securId
5. check list of items to see **global-read-only** policy
6. go to the policy details and check that form is non-editable

### Links
[JIRA](https://hashicorp.atlassian.net/browse/CC-5792)
[Figma](https://www.figma.com/file/oMJ1VqcRzb2EP1mqfaohDz/View-Builtin-Token?type=design&node-id=916-8806&mode=design&t=ZRNWWdS4Bb0fhced-0)

### PR Checklist

* [x] updated test coverage
* [ ] external facing docs updated
* [x] appropriate backport labels added
* [x] not a security concern

### CODE REVIEW

File: ui/packages/consul-ui/app/components/consul/policy/list/index.hbs [hunk 1]
```diff
@@ -8,13 +8,19 @@
...
-            </Tooltip>
+            {{#if (eq (policy/typeof item) 'policy-management')}}
+              <Tooltip>
+                Global Management Policy
+              </Tooltip>
+            {{else}}
+              <Tooltip>
+                Global Read-only Policy
+              </Tooltip>
+            {{/if}}
```
Comment (chris-hut) [inline ui/packages/consul-ui/app/components/consul/policy/list/index.hbs:23]: Was going to suggest maybe we could simplify this a bit by doing most of this logic in the component

```js
get specialPolicyTooltip() {
  if (policyTypeOfItem('policy-management')) {
     return 'Global Management Policy';
  }

  if (policyTypeOfItem('read-only')) { 
    return 'Global Read-only policy';
  }
}
```

But I don't even see a component, I'm not sure how to call this helper outside of here, and this is the core-ui 😅, so feels not worth trying to change!

Just a thought though, definitely not blockable!

**Review by chris-hut — APPROVED** [review]
I'm even more scared of `consul/ui` than I was before, but everything here makes sense!

**Comment by hc-github-team-consul-core** [comment]
Backport failed @valeriia-ruban. Run: https://github.com/hashicorp/consul/actions/runs/6151430456

**Comment by hc-github-team-consul-core** [comment]
Backport failed @valeriia-ruban. Run: https://github.com/hashicorp/consul/actions/runs/6151430464

**Pull request merged**